client/network/src/protocol/sync.rs

```diff
@@ -1316,18 +1412,19 @@ impl<B: BlockT> ChainSync<B> {
 		}
 
 		if let PeerSyncState::AncestorSearch {..} = peer.state {
+			trace!(target: "sync", "Peer state is ancestor search.");
```

Comment:
This message could use more context, as it may be interleaved with other messages in the log. Perhaps would be better to additionally print `peer.state` [here](https://github.com/paritytech/substrate/pull/7733/files#diff-a5dfc701556975f194fd5d7df78d855b7d2007e741b565025022b849d143c4eeR1372) instead

Reply:
At the top of the function we already print the peer id and nothing else can be printed in between from syncing that isn't related to this peer. So, we don't need to print anymore here.